README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/FCO/Red.svg?branch=master)](https://travis-ci.org/FCO/Red)
+[![Build Status](https://travis-ci.org/Altreus/Red.svg?branch=master)](https://travis-ci.org/Altreus/Red)
```

Comment:
This is probably a thinko.

Reply:
Ah, that'll be `mi6` being clever

Reply:
Should I just remove the README commit and only PR the POD?

Reply:
That's for @FCO to decide, I just pointed the thing out so it won't be accidentally merged. :+1:

Reply:
Thank you for noticing! :)